src/Slices/Login/UI/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./LoginForm";
```

Comment:
Why do we need to export this twice? There are two index files with the same export statement

Reply:
I put the component into own folder, I will flatten the structure